lib/Parser/OglVersion.php

```diff
@@ -63,21 +63,44 @@ public function getGlslVersion() {
         return $this->glslVersion;
     }
 
-    // GL/GLSL extensions.
+    /**
+     * Add an extension, or merge it if it already exists.
+     *
+     * @param string $name Name of the extension.
+     * @param string $status Status of the extension.
+     * @param array $supportedDrivers List of drivers supported for this extension.
+     * @param Mesamatrix\Git\Commit $commit The commit used by the parser.
+     *
+     * @return Mesamatrix\OglExtension The new or existing extension.
```

Comment:
Put a backslash `\` in front of those namespaced classes, otherwise static code analysers will break.